internal/cloudrunci/gcloud_test.go

```diff
@@ -27,21 +29,36 @@ func TestCreateIDToken(t *testing.T) {
 	// TODO assign to token
 	_, err := CreateIDToken("http://example.com")
 	if err != nil {
-		t.Errorf("CreateIDToken: %q", err)
+		t.Errorf("CreateIDToken: %v", err)
 	}
 
 	// validate token
 }
 
 func TestGcloud(t *testing.T) {
 	testutil.EndToEndTest(t)
-	out, err := gcloud("label", exec.Command(gcloudBin, "help"))
+	out, err := gcloudWithoutRetry("success", exec.Command(gcloudBin, "help"))
 	if err != nil {
-		t.Errorf("CreateIDToken: %q", err)
+		t.Errorf("gcloud: %v", err)
 	}
 
 	want := "gcloud - manage Google Cloud Platform resources and developer workflow"
 	if got := string(out); !strings.Contains(got, want) {
 		t.Errorf("gcloud: got (%s), want (%s)", got, want)
 	}
 }
+
+func TestGcloudRetry(t *testing.T) {
+	originalWriter := log.Writer()
```

Comment:
This was added in Go 1.13, so this is failing to compile with Go 1.11.

https://golang.org/doc/go1.13#log

I think we can assume `os.Stderr`.

Reply:
Somehow I keep forgetting this limitation. Thanks for patiently flagging it each time.